iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/gcp/HighPrivilegedRolesOnWorkloadResourcesCheck.java

```diff
@@ -29,5 +29,6 @@
   @Override
   public void initialize(InitContext init) {
     new HighPrivilegedRolesOnIamResourcesCheckPart().initialize(init);
+    new HighPrivilegedRolesOnCloudIdentityGroupCheckPart().initialize(init);
   }
```

Comment:
As you already realized. There is no need for this "umbrella" check class. Please merge both check parts into this class.